ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -905,6 +905,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts
+- as: aws-rosa-sts-hypershift-p2-f7
```

Comment:
Would you mind to add the test to `release-4.12__amd64-stable.yaml`, too ?

Reply:
@yasun1 Do we need to configure `stable`, because aws-rosa-sts-p2-f7 only has `nightly`, so I don't know if we need to configure it

Reply:
No. Now we only do the OCP e2e on the nightly rosa hypershift cluster.

Reply:
Ack